test/integration/test_abstract_class_doubling.gd

```diff
@@ -0,0 +1,70 @@
+extends GutInternalTester
+
+
+@abstract
+class AbstractClass:
+
+	@abstract func abstract_method()
+
+
+@abstract
+class ExtendsAbstractClass:
+	extends AbstractClass
+
+	func abstract_method():
+		return "implemented"
+
+	@abstract
+	func another_abstract_method()
+
+
+func before_all():
+	register_inner_classes(get_script())
+
+func test_can_double_abstract():
+	var dbl = double(AbstractClass)
+	assert_not_null(dbl)
+
+
+func test_can_stub_to_return_for_abstract_method_at_sctipt_level():
+	stub(AbstractClass, 'abstract_method').to_return('a')
+	var inst = double(AbstractClass).new()
+	assert_eq(inst.abstract_method(), 'a')
+
+
+func test_can_stub_to_return_for_abstract_method_at_double_level():
+	var Dbl = double(AbstractClass)
+	stub(Dbl, 'abstract_method').to_return(9)
+	var inst = Dbl.new()
+	assert_eq(inst.abstract_method(), 9)
+
+
+func test_can_stub_to_return_for_abstract_method_at_instance_level():
+	var inst = double(AbstractClass).new()
+	stub(inst.abstract_method).to_return(7)
+	assert_eq(inst.abstract_method(), 7)
+
+
+func test_error_when_stubbing_to_call_super_at_script_level():
+	stub(AbstractClass, 'abstract_method').to_call_super()
+	pass_test("no errors")
```

Comment:
Not sure how to turn this into a meaningful test :)

Reply:
replace `pass_test` with `assert_tracked_gut_error(gut, 0)`.  Now that we know what kind of error is thrown we can test for it.